samples/drivers/led/pwm/Kconfig

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2024 TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+config BLINK_DURATION_SHORT
+	int "LED on and off times for short-cycle blinking demo"
+	default 100
+	help
+	  Specifies the LED blinking time in milliseconds in the short-cycle
+	  blinking demo of the PWM-LED sample. If set to 0, the short-cycle
+	  blinking demo will not be performed.
+
+config BLINK_DURATION_LONG
+	int "LED on and off times for long-cycle blinking demo"
+	default 1000
+	help
+	  Specifies the LED blinking time in milliseconds in the long-cycle
+	  blinking demo of the PWM-LED sample. If set to 0, the long-cycle
+	  blinking demo will not be performed.
+
+config FADE_DELAY
+	int "delay time for fade demo"
+	default 10
+	help
+	  Specifies the delay time for the fade demo of the PWM-LED sample.
```

Comment:
Suggestion: Specifies the delay in milliseconds for ...

Reply:
I applied it.